pkg/ebpf/tracee.go

```diff
@@ -913,6 +912,15 @@ func (t *Tracee) initBPF() error {
 		return err
 	}
 
+	err = t.attachSelfUprobe("uprobe_syscall_trigger", "github.com/aquasecurity/tracee/pkg/ebpf.(*Tracee).invokeTriggeredSyscalls")
```

Comment:
These probes should be integrated into the probes package.

Reply:
https://github.com/aquasecurity/tracee/pull/2031#discussion_r936178427